src/string.rs

```diff
@@ -286,6 +286,11 @@ unsafe extern "C" fn one_byte_const_estimate_memory_usage(
 ) -> int {
   -1
 }
+unsafe extern "C" fn one_byte_const_estimate_shared_memory_usage(
+  _this: *const OneByteConst,
+  _recorder: *mut (),
+) {
+}
```

Comment:
This function does nothing? 😕

Reply:
it doesn't need to do anything, it just needs to exist for the vtable shape to be correct